src/OpenApiV3Validator.php

```diff
@@ -240,4 +242,14 @@ private function setContentType(string $contentType): self
 
         return $this;
     }
+
+    /**
+     * @param string $responseCode
+     *
+     * @return bool
+     */
+    private function emptyResponseExpected($responseCode)
+    {
+        return 204 !== $responseCode;
```

Comment:
This seems to be doing the opposite of what the function name says. It returns true if it is not a 204, ie we aren't expecting an empty response. Also add the bool return type to the method signature